doc/developers/develop.rst

```diff
@@ -538,19 +538,16 @@ allow_nan (default=False)
 array_api_support (default=False)
     whether the estimator supports Array API compatible inputs.
 
-binary_only (default=False)
-    whether estimator supports binary classification but lacks multi-class
-    classification support.
-
-multilabel (default=False)
-    whether the estimator supports multilabel output
-
-multioutput (default=False)
-    whether a regressor supports multi-target outputs or a classifier supports
-    multi-class multi-output.
-
-multioutput_only (default=False)
-    whether estimator supports only multi-output classification or regression.
+target_type (default=["multiclass", "single-output"])
```

Comment:
If the estimator is a regressor, then having a "multiclass" tag feels strange to me.